bubble_sort.rb

```diff
@@ -0,0 +1,35 @@
+def bubble_sort(array)
+    swap = true
+    while swap do
+        swap = false
```

Comment:
Tip for Future: Convention in Ruby is to use 2 space indentation!

Reply:
Seems you are using 4

Reply:
Okay, Noted. 
Thank you.